src/reducers/__tests__/query.spec.jsx

```diff
@@ -711,6 +711,38 @@ describe( 'reducer:query', () => {
       } )
     } )
 
+    describe( 'DATA_LENS_CHANGED actions', () => {
+      it( 'changes the dataLens', () => {
+        const action = {
+          type: actions.DATA_LENS_CHANGED,
+          dataLens: 'foo'
+        }
+        const result = target( {}, action )
+        expect( result ).toEqual( {
+          dataLens: 'foo',
+          enablePer1000: true,
+          mapWarningEnabled: true,
```

Comment:
May want to think about this.  It's not really a filter, so it shouldn't affect the per1000 warning when they toggle back to the map tab